src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -229,7 +240,7 @@ await foreach (ConfigurationSetting setting in _client.GetConfigurationSettingsA
                         LabelFilter = loadOption.LabelFilter
                     };
 
-                    // Load all key-values with the null label.
+                    // Load all key-values with the specified label.

```

Comment:
>// Load all key-values with the specified label. [](start = 20, length = 48)

We could just remove this comment since it could be confusing. We aren't loading all key-values here, just the ones corresponding to one selector.